alt-integration-grpc-service/conf/webservice-default.properties

```diff
@@ -3,3 +3,31 @@ apiPort=19011
 
 # Veriblock network
 veriblockNetwork=main
+
+# AltChainParameters.keystoneInterval=int
+
+# Forkresolution.keystoneFinalityDelay=int
+# Forkresolution.keystoneFinalityDelay=int
+
+# PopRewardCurveConfig.startOfDecreasingLine=decimal
+# PopRewardCurveConfig.widthOfDecreasingLineNormal=decimal
+# PopRewardCurveConfig.widthOfDecreasingLineKeystone=decimal
+# PopRewardCurveConfig.aboveIntendedPayoutMultiplierNormal=decimal
+# PopRewardCurveConfig.aboveIntendedPayoutMultiplierKeystone=decimal
```

Comment:
PopRewardCurveConfig - I think "Config" part is not required here. Better is to make it "PopRewardCurve".
The same for other "Config" fields above and below.